core/src/main/java/io/smallrye/context/SmallRyeContextManager.java

```diff
@@ -248,6 +251,20 @@ public SmallRyeContextManager build() {
         //
         // Extras
 
+        /**
+         * Make all created {@link SmallRyeManagedExecutor} forward to the given executor service by default instead of
+         * creating new executor services to back them. This can be overridden with
+         * {@link SmallRyeManagedExecutor.Builder#withExecutorService(ExecutorService)} and
+         * {@link SmallRyeManagedExecutor.Builder#withNewExecutorService()}. Also serves as the default executor to use
+         * by all {@link CompletionStage} and {@link CompletableFuture} wrapped by {@link ThreadContext}.
+         * 
+         * @param executorService the executor service to delegate to. If <code>null</code>, all created {@link ManagedExecutor}
+         *        will create new backing executor services, and all <code>*Async</code> methods of the
+         *        {@link CompletionStage} and {@link CompletableFuture} wrapped by {@link ThreadContext} will throw due to a
+         *        lack
+         *        of executor.
```

Comment:
Unnecessary newline? BTW the same is repeated throughout the javadoc of `SmallRyeManagedExecutor` as well, probably some auto-formatting?

Reply:
Done.